src/resources/ui/preferences.ui

```diff
@@ -80,7 +108,7 @@
     <object class="GtkAboutDialog" id="about-dialog">
         <property name="program-name">GNOME RunCat</property>
         <property name="authors">Sergei Kolesnikov https://kolesnikov.se/</property>
-        <property name="artists">Sergei Kolesnikov, Takuto Nakamura</property>
+        <property name="artists">exincore, Sergei Kolesnikov, Takuto Nakamura, WebSnke</property>
```

Comment:
Hello! While translating is an art, you may be more accurately credited as a translator with the "translator-credits" property.

The following line should work when inserted alongside the other credits.
```xml
<property name="translator-credits">WebSnke</property>
```

Reply:
Thanks, I just pushed the suggestion.

Reply:
You did it wrong the second time.
a. https://github.com/win0err/gnome-runcat/pull/40#issuecomment-1469577556
b. https://github.com/win0err/gnome-runcat/pull/41#discussion_r1148617612

Please, be more attentive

Reply:
Ok, I just changed it to "translator-credits".

Reply:
Please, fix listing (`npm run lint --fix`) and I'll merge the PR

Reply:
> Please, fix linting (`npm run lint --fix`) and I'll merge the PR

NodeJS is not currently installed on my system. Could you do it instead, please?

Reply:
> > Please, fix linting (`npm run lint --fix`) and I'll merge the PR
> 
> NodeJS is not currently installed on my system. Could you do it instead, please?

I just downloaded NodeJS and tried to use your command but it did not work. After a bit of research, I found out that it is just a warning and could be ignored.

Reply:
I think adding a command to update gettext to the main.yml file could resolve this issue.

Reply:
I'm sorry, check out the output of CI run: https://github.com/win0err/gnome-runcat/actions/runs/4541393529/jobs/8003501586?pr=41 

There's a problem with translations that are not up-to-date. Please, try to update them: `make translations`

Reply:
It seems like you need to update some sort of ITS rule file to remove the warning.

Reply:
Are you serious? Why I should remove checks that I set up to check that translations are up-to-date and extension's code is well-formatted?